### ISSUE TITLE
FileNotFoundError

### ISSUE THREAD

**Issue by adeepH** [dmlc/gluon-cv#1409]
After installing gluonCV on anaconda as given in the installation guide, I tried running a sample code.
![Annotation 2020-08-08 125019](https://user-images.githubusercontent.com/46108405/89704879-c5229380-d975-11ea-8a5a-b5ee256adc57.png)

**Comment by Neutron3529** [comment]
Which file?

put the full traceback here would be more helpful for finding the correct error.

IMO, it might due to the incorrect installation of mxnet.

can you execute `import mxnet as mx`?
If you could, import `gluoncv` after import `mxnet` may be helpful.

**Comment by adeepH** [comment]
I have installed the stable versions of both `mxnet` and `gluoncv`
But I'm still facing this error
![Filenpt](https://user-images.githubusercontent.com/46108405/89854348-56ca1500-dbb1-11ea-9274-92629a4d035f.PNG)

**Comment by zhreshold** [comment]
seems like installation of mxnet is not found.
Can you check the path 'C:\xxxx\libmxnet.dll' in the log exists? Otherwise it might due to corrupted installation.
Try uninstall mxnet and install one using one of these: https://dist.mxnet.io/python/cpu
For example 

```
pip install https://repo.mxnet.io/dist/python/cpu/mxnet-1.6.0-py2.py3-none-win_amd64.whl
```

**Comment by dai-ichiro** [comment]
I think the problem can be solved using Python version 3.7.
